stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -80,7 +80,6 @@ spec:
         imagePullPolicy: {{ .Values.artifactory.image.pullPolicy }}
         securityContext:
           allowPrivilegeEscalation: false
-      {{- if .Values.artifactory.preStartCommand }}
```

Comment:
@danielezer Can you please add this line.

Reply:
Nice catch. sorry about that. Thanks